.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "venv/bin/python3.6"
+}
```

Comment:
Could you add 
```
# vscode
.vscode/**
```

to `.gitignore` instead of adding `.vscode/settings.json`? Thanks!

Reply:
done !